ops/pebble.py

```diff
@@ -3211,6 +3317,44 @@ def get_notices(
         resp = self._request('GET', '/v1/notices', query)
         return [Notice.from_dict(info) for info in resp['result']]
 
+    def get_identities(self) -> Dict[str, Identity]:
+        """Get all identities in Pebble.
+
+        Returns:
+            A dict mapping identity names to :class:`Identity` objects.
+        """
```

Comment:
We should add in a `.. jujuadded:: 3.6.x` with the right version here and for `replace_identities` and `remove_identities` - I don't think we need that for any of the typed dicts/dataclasses, just for the three `Client` methods.